documentation/en/source/pages/migration/migrations-spec/overview.rst

```diff
@@ -0,0 +1,17 @@
+.. Copyright (c) 2007-2018 UShareSoft, All rights reserved
+
+.. _migrations-spec:
+
+Migrations Specification
+========================
+
+Migrations contain all the information used to create migrations. They are JSON or YAML files, passed as a parameter to the hammr command-line.
+
+A migration has the following part:
+
+* ``migration``: defines the migration.
+
+.. toctree::
+   :titlesonly:
+
+   migration
```

Comment:
Should not be the reference to builders part here? Or elsewhere? (I cannot find it) Maybe a migration-target file?

Reply:
agree